### DISCUSSION TITLE
General Chat - LinkFree CLI

### DISCUSSION THREAD

**Discussion by Pradumnasaraf** [Pradumnasaraf/BioDrop-CLI#41] [General]
Hey, welcome! This is a general discussion, so feel free to chat about anything related to LinkFree CLI. 🎉

**Comment by Ananya2001-an** [comment]
Hey @Pradumnasaraf it will be a good idea to have a blog that explains how to use LinkFree CLI from start till end. I am thinking of writing it. WDYT? We can then add it in the readme as well.

**Reply by Pradumnasaraf** [reply]
Hey, @Ananya2001-an 

I love the idea, but here are my two cents. I know writing this will take lots of effort, but as the project is not standalone and people will directly use docs in the LinkFree to see the use it (Not come here to see how to use it), I think it makes very less sense to add it here. Also, we already have docs in Markdown (in LinkFree Docs), I think the better option would be to improve that. I don't wan't make your effort to get unseen. 

Hope you get my thoughts. Love to know WDYT

**Reply by Ananya2001-an** [reply]
Oh yes you are right, I forgot about the LinkFree docs. My bad. No issues I understand, this will be a wasted effort indeed.

**Reply by Pradumnasaraf** [reply]
Thank you for understanding.
